documentation/modules/auxiliary/gather/mikrotik_winbox_fileread.md

```diff
@@ -0,0 +1,60 @@
+## Vulnerable Application
+
+MikroTik RouterOS allows unauthenticated remote attackers to read arbitrary files
+through a directory traversal through the WinBox interface (typically port 8291).
+
+Vulnerable versions of MikroTik RouterOS:
+
+* (bugfix) 6.30.1-6.40.7
+* (current) 6.29-6.42
+* (RC) 6.29rc1-6.43rc3
```

Comment:
@adfoster-r7 MikroTik used to publish three versions of their firmware: bugfix, current, and release candidate. Now they publish four: long-term, stable, test, and dev: https://mikrotik.com/download/

Unfortunately, I think @h00die has made a best effort here because MT's versioning is a bit weird. For example, 6.40.9 (longterm / bugfix) is **not** vulnerable to this bug, but 6.40.5 (current / stable) and 6.41.3 (current / stable) **are** vulnerable.  You basically have to refer to each branch when discussing a vulnerability and when/if it got fixed.

Reply:
I had a response typed up, then had to deal with some IRL stuff, and @jacob-baines beat me to it.  They would DEFINITELY know better than me!